src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -2854,6 +2854,45 @@ private static void RenameElementToStandardDirectory(XElement element)
             }
         }
 
+        // If TryGetInnerText succeeds, normalize the string value it returns by replacing all \r's \n's with spaces,
+        // then replacing all multiple spacws with single spaces. This is suitable for use in an XText representing a condition.
+        private static bool TryGetNormalizedInnerText(XElement element, out string value, out List<XNode> comments)
+        {
+            var found = TryGetInnerText(element, out var localValue, out comments);
+
+            if (found && null != localValue)
+
+            {
+                var whiteSpace = false;
+                var sb = new StringBuilder();
+
+                foreach (var c in localValue)
+                {
+                    if ('\u0020' == c || '\u000a' == c || '\u000d' == c)
```

Comment:
This is the complicated way of saying:

```C#
if (' ' == c || '\r' == c || '\n' == c)
```

Right? I think this is the wrong way to solve the problem (see comment above about Foo and Bar) but using Unicode values seems overkill, no?

Reply:
Certainly, the Unicode encoding is overkill. I used it to emphasize the literal result of passing the literal characters through the XText constructor. This code is likely to disappear once I decide how to deal with the issue above.